plugins/producer_plugin/include/eosio/producer_plugin/subjective_billing.hpp

```diff
@@ -114,7 +114,7 @@ class subjective_billing {
 public:
    void disable() { _disabled = true; }
    void disable_account( chain::account_name a ) { _disabled_accounts.emplace( a ); }
-   bool is_account_disabled(const account_name& a ) const { return _disabled_accounts.count( a ); }
+   bool is_account_disabled(const account_name& a ) const { return _disabled || _disabled_accounts.count( a ); }
```

Comment:
Why is this change here?  It looks right, but seems unrelated.

Reply:
It is unrelated. I will pull it out into a diff PR.